pkg/services/notifications/notifications_test.go

```diff
@@ -31,7 +31,7 @@ func TestNotifications(t *testing.T) {
 			err := sendResetPasswordEmail(&m.SendResetPasswordEmailCommand{User: &m.User{Email: "asd@asd.com"}})
 			So(err, ShouldBeNil)
 			So(sentMsg.Body, ShouldContainSubstring, "body")
-			So(sentMsg.Subject, ShouldEqual, "Reset your Grafana password - asd@asd.com")
+			So(sentMsg.Subject, ShouldEqual, "Reset your raintank password - asd@asd.com")
```

Comment:
This is not right, will cause merge conflict later, I made the email system so all templates can be replaced for raintank without modifying the grafana templates. they can be in their own path, no grafana templates should be modified in raintank branch!